### PR TITLE
Importing queries with mg_client

### PR DESCRIPTION
**Pull request by mculinovic**

### CODE REVIEW

File: user_technical/how_to_guides/import-data.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
 ## How to Import Data?
 
 Memgraph comes with tools for importing data into the database. Currently,
-only import of CSV formatted is supported. We plan to support more formats in
-the future.
+only [import of CSV](#import-csv) formatted is supported. We plan to support more formats in
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: formatted -> format

File: user_technical/how_to_guides/import-data.md [hunk 2]
```diff
@@ -1,10 +1,13 @@
...
+the future. Furthermore, [cypher queries](#import-cypher) saved in a file can be imported with
+`mg_client` running in non-interactive mode.
 
-### CSV Import Tool
+
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: Delete newline.

File: user_technical/how_to_guides/import-data.md [hunk 3]
```diff
@@ -48,6 +51,7 @@ When using Docker, this translates to:
 docker run --entrypoint=mg_import_csv memgraph --help
 ```
 
+
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: Delete newline.

File: user_technical/how_to_guides/import-data.md [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
-only import of CSV formatted is supported. We plan to support more formats in
-the future.
+only [import of CSV](#import-csv) formatted is supported. We plan to support more formats in
+the future. Furthermore, [cypher queries](#import-cypher) saved in a file can be imported with
+`mg_client` running in non-interactive mode.
 
-### CSV Import Tool
+
+
+### CSV Import Tool {#import-csv}
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: Repeated title?

File: user_technical/how_to_guides/import-data.md [hunk 5]
```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
...
 
 Next time you run Memgraph, the dataset will be loaded.
+
+### Importing Cypher Queries {#import-cypher}
+
+When Memgraph is running, cypher queries are imported by running `mg_client` in
+non-interactive mode. The user can import queries saved in e.g. `queries.txt`
+by issuing the following shell command:
+
+`mg_client < queries.txt`
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: (view raw comment)

```bash
mg_client < queries.txt
```

This is not noticeable on github but its important for syntax highlighting on the web.

File: user_technical/how_to_guides/import-data.md [hunk 6]
```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
...
+
+If you installed Memgraph using Docker, you will need to run the client using
+the following command:
+
+`docker run -i --entrypoint=mg_client memgraph --host HOST < queries.txt`
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: See previous comment.

File: user_technical/how_to_guides/import-data.md [hunk 7]
```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
...
+
+Remember to replace `HOST` with valid IP of the container(see
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: space between container and open parenthesis. This seems like you're calling the container function :)

File: user_technical/how_to_guides/import-data.md [hunk 8]
```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
...
+[Note for OS X/macOS Users](../tutorials/quick-start.md#OSX-note)).
+
+For more information about `mg_client` options run:
+
+`mg_client --help`
+
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: Delete newline.

File: user_technical/how_to_guides/import-data.md [hunk 9]
```diff
@@ -116,3 +120,24 @@ docker run -v mg_lib:/var/lib/memgraph -v mg_etc:/etc/memgraph -v mg_import:/imp
...
+the following command:
+
+`docker run -i --entrypoint=mg_client memgraph --host HOST < queries.txt`
+
+Remember to replace `HOST` with valid IP of the container(see
+[Note for OS X/macOS Users](../tutorials/quick-start.md#OSX-note)).
+
+For more information about `mg_client` options run:
+
+`mg_client --help`
```
Comment (ipaljak) [inline user_technical/how_to_guides/import-data.md]: (view raw comment)

```bash
mg_client --help
```

File: user_technical/how_to_guides/import-data.md (hunk 4 above)
Comment (mculinovic) [reply user_technical/how_to_guides/import-data.md]: No, only link added.

**Pull request merged**